backend/layers/business/business_interface.py

```diff
@@ -143,6 +160,9 @@ def get_latest_published_collection_versions_by_schema(
     ) -> List[CollectionVersionWithPublishedDatasets]:
         pass
 
+    def _delete_from_bucket(self, bucket: str, keys: List[str], prefix: str) -> None:
```

Comment:
you don't need to include private functions in the interface definition

Reply:
Don't need to... or shouldn't? 🤔 

Perhaps the latter so that the places where business logic is instantiated as BusinessInterface will not prompt the developer to call that private method... removing it now